tests/experimental/datasources/test_pandas_datasource.py

```diff
@@ -136,7 +136,7 @@ def test_data_asset_defined_for_io_read_method(self, method_name: str):
         ["asset_model", "extra_kwargs"],
         [
             (CSVAsset, {"sep": "|", "names": ["col1", "col2", "col3"]}),
-            (JSONAsset, {"orient": "records", "encoding_errors": "strict"}),
+            (JSONAsset, {"orient": "records", "convert_dates": True}),
```

Comment:
Note that `encoding_errors` does not exist on pandas `1.1`
https://pandas.pydata.org/pandas-docs/version/1.1/reference/api/pandas.read_json.html#pandas.read_json